### DISCUSSION TITLE
Topics with latest ui change

### DISCUSSION THREAD

**Discussion by postlund** [community/community#22059] [New to GitHub]
<p>After the latest ui change, how do I update (or even display) topics for my projects? Specifically on mobile devices, safari on iOS in my case. It’s seems like it’s not displayed anymore.</p>

**Comment by yamiacat** [comment]
<p>Hi <a class="mention" href="/postlund">@postlund</a>!</p>
<p>Topics appear in the “About” section of the sidebar on the right side of the repository front screen now.</p>
<p>You can update and add them by clicking the cog wheel symbol next to “About”:</p>
<p><a href="https://docs.github.com/en/github/administering-a-repository/classifying-your-repository-with-topics#adding-topics-to-your-repository" class="onebox" target="_blank">https://docs.github.com/en/github/administering-a-repository/classifying-your-repository-with-topics#adding-topics-to-your-repository</a></p>

**Comment by postlund** [comment]
<p>Hey <a class="mention" href="/yamiacat">@yamiacat</a>!</p>
<p>That section is never rendered on a mobile device, no matter if the desktop site is requested or mobile pages disabled completely. I guess it has to do with the responsive design somehow? I managed to change (and view) topics from an iPad though, so I fixed it in the end. But I would prefer that it worked on my phone as well.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi <a class="mention" href="/postlund">@postlund</a> - Ah, I see!</p>
<p>Sorry, I did a quick test on an iPad.</p>
<p>Yep, I’ll report this, as you can’t seem to access those settings on the mobile site from any browser!</p>

**Comment by postlund** [comment]
<p>Perfect, thank you! :+1:</p>

**Comment by Thienanbad** [comment]
Still not fix
